lib/sync-settings.coffee

```diff
@@ -232,6 +232,7 @@ SyncSettings =
             if atom.config.get('sync-settings.syncPackages')
               callbackAsync = true
               @installMissingPackages JSON.parse(file.content), cb
+              @uninstallRedundantPackages JSON.parse(file.content), cb
```

Comment:
While this is a great feature I think it is not necessarily what all users want. Therefore I think this should be an option in the settings which defaults to `false` to maintain the current behavior.